### PR TITLE
Cherry pick address

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: test/unit/qrda/patient_round_trip_test.rb [hunk 1]
```diff
@@ -55,6 +68,32 @@ def generate_shell_patient(type)
         cqm_patient
       end
 
+      def test_interval_round_trips
```
Comment (lmd59) [inline test/unit/qrda/patient_round_trip_test.rb:71]: This method was a part of commit "Allow for low and high times to be nullFlavored" and not a part of the patient addresses branch. Do we want to include it here?
